eng/scripts/Update-PkgVersion.ps1

```diff
@@ -91,9 +91,15 @@ if ($packageSemVer.HasValidPrereleaseLabel() -ne $true){
 }
 
 if (!$packageOldSemVer.IsPrerelease -and ($packageVersion -ne $NewVersionString)) {
+  $whitespace = $propertyGroup["Version"].PreviousSibling
   if (!$propertyGroup.ApiCompatVersion) {
     $propertyGroup.InsertAfter($csproj.CreateElement("ApiCompatVersion"), $propertyGroup["Version"]) | Out-Null
-    $whitespace = $propertyGroup["Version"].PreviousSibling
+    $propertyGroup.InsertAfter($whitespace.Clone(), $propertyGroup["Version"]) | Out-Null
+  }
+  $ApiCompatVersionComment = "The ApiCompatVersion will be updated automatically and does not need to be modified."
```

Comment:
I intentionally didn't make it say that it shouldn't be changed because there are valid cases where someone may want to change it - e.g. pinning to a preview version. The goal of the comment is to answer the question - "Do I need to change this?"

Reply:
How about "The ApiCompatVersion is managed automatically and should not generally be modified manually."

Reply:
Works for me.